modules/configuring-access-control-and-sending-invitations.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * configuration/inviting-users-to-rhacs.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-access-control-and-sending-invitations_{context}"]
+= Configuring access control and sending invitations
+
+.Procedure
+
+. To invite a user, use one of the following methods:
```

Comment:
Just because there are two ways of getting there doesn't mean that you have to tell users the available options. It's usually better to just pick one of them and go with that. I'd pick whichever one is easiest/more likely for people to use.

Reply:
@vjwilson any thoughts?

Reply:
It's a toss-up about which is easier:
- if they've just added an auth provider to the system, then the button on the auth provider page is more obvious
- anytime they invite users after that, the option in the User Profile menu will be easier

In this case, I don't think there is an obvious one to choose, so I would leave both in the doc.

Reply:
@vjwilson Thanks and appreciate your suggestion. However, from a documentation perspective, we would take a minimalist approach by documenting the happy path that is most obvious and easiest for the customer. 
Would you consider it appropriate to document only the path where the customer navigates to the *Platform Configuration - Access Control - Auth providers* tab and then clicks on *Invite users* since it is very obvious that the customer would create an authentication provider?

Reply:
I respectfully disagree with the docs teams heuristic in this case, but if you simply cannot bend from that rule, then your choice is as good as the other one.

Reply:
Thank you for sharing your perspective. I highly value your input, and feedback is highly appreciated.